### ISSUE TITLE
Separate Artist / Track / Label files

### ISSUE THREAD

**Issue by Fr0sti2018** [Unreal-Dan/RekordBoxSongExporter#4]
First off: this program is awesome, so thank you very much for it!

Now my feature request, maybe this can be done very quickly for pro:
In OBS I can add the current_track.txt and I have _one_ line with
%artist% -  %title% -  %label% 

I would like to have in that format (which i can format separetly!):
%artist%
%title%
%label%

With a line break. Unfortunetly OBS can't detect a line break. So my suggestion would be that RekordBoxSongExporter would generate each information additional, like:
current_artist.txt --> %artist%
current_title.txt --> %title%
current_label.txt --> %label%

So you can add "n" OBS text sources. In this example 3, artist, title and label.
It would be awesome, if you can make that happen.

Alternatively one file current_track_alternative.txt with line break:
%artist%
%title%
%label%
...

Thanks for your time!

**Comment by Unreal-Dan** [comment]
I want to make a more sophisticated templating system in the future but haven't gotten around to it. I appreciate your comments and I will try to get an update out soon to address this.

In the meantime if you want to try building from source with visual studio 2019 it should build without any issues, it requires no dependencies.

All you need is visual studio community edition 2019, it's free from microsoft.

If you add in some code around here https://github.com/Unreal-Dan/RekordBoxSongExporter/blob/master/Module/OutputFiles.cpp#L256

That looks like:

```
        // write out custom track files
        rewrite_file(get_dll_path() + "title.txt", entry.title);
        rewrite_file(get_dll_path() + "artist.txt", entry.artist);
        rewrite_file(get_dll_path() + "label.txt", entry.label);
```

You can write out your own track files with whatever information you want, I don't think any other changes would be necessary.

Also, make sure you select release mode when you build the project.

**Comment by Fr0sti2018** [comment]
Thanks for the fast reply, much appreciated!
I've downloaded the latest source and could successfully compile it with Visual Studio 2019 (Community Edition).

The code has do be modified a little:
rewrite_file(get_dll_path() + "\\"+ "artist.txt", entry.artist);

I made plenty of other stuff for my own, tried different things, e.g. out remove of keys and BPMs in titles before writing the text file(s). Works, I'm pretty happy now...it's not professional / good coded, but I'm happy ;)
